code/modules/surgery/bodyparts/robot_bodyparts.dm

```diff
@@ -514,6 +514,44 @@
 
 	biological_state = (BIO_METAL|BIO_JOINTED)
 
+// Advanced Limbs: More durable, high punching force
+
+/obj/item/bodypart/arm/left/robot/advanced
+	name = "advanced robotic left arm"
+	desc = "An advanced cybernetic arm, capable of greater feats of strength and durability."
+	unarmed_damage_low = 5
+	unarmed_damage_high = 13
+	unarmed_effectiveness = 20
+	max_damage = 75
+	body_damage_coeff = 0.50
```

Comment:
You should include a comment explaining the whole "more max health, less coeff" thing.

Also this seems easy to break if the default value is changed in the future.

Reply:
Maybe I should make a define for these + limb baseline, similar to prosthetics? As well as for the coefficient value?

Reply:
It's prooooobably fine.

Reply:
Let's not take any chances haha